presto-base-jdbc/src/main/java/io/prestosql/plugin/jdbc/CachingJdbcClient.java

```diff
@@ -89,14 +107,14 @@ public boolean schemaExists(ConnectorSession session, String schema)
     @Override
     public Set<String> getSchemaNames(ConnectorSession session)
     {
-        JdbcIdentity key = JdbcIdentity.from(session);
+        SchemaNamesCacheKey key = new SchemaNamesCacheKey(JdbcIdentity.from(session), getSessionProperties(session));
```

Comment:
This seems redundant. I think we can skip session props here for now.

Reply:
This was raised by @kokosing so I've added this for consistency

Reply:
@kokosing this seems redundant

Reply:
1. I don't want to uncover this in few months, that this was implemented only for certain places.
2. I don't want to remember about this corner case.
3. I don't know all custom connectors that are out there.

Assuming  https://github.com/prestosql/presto/pull/6167/commits/eb3fc505b9b18d40fa019411a31f1fd2e07d0b57#r534067384, we should not be worried. Notice that we cache only operations that could take much longer.